test/language/for-of/control-flow/break-label-from-catch.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 13.6.4.13 S5.n
+description: >
+    Control flow during body evaluation should honor `break` statements within
+    `try` blocks.
+---*/
+
+function* values() {
+  yield 1;
+  yield 1;
```

Comment:
Sounds good to me. Additionally, the second `yield` statement is extraneous. This was a pattern in a number of tests, so I've fixed it in all cases.